src/app/accounts/active-accounts.ts

```diff
@@ -0,0 +1,38 @@
+import { MS_PER_30_DAYS, USER_ACTIVENESS_MONTHLY_VOLUME_THRESHOLD } from "@config/app"
+import { LedgerService } from "@services/ledger"
+import { AccountsRepository } from "@services/mongoose"
+
+// user is considered active if there has been one transaction of more than USER_ACTIVENESS_MONTHLY_VOLUME_THRESHOLD sats in the last 30 days
+const isAccountRecentlyActive = async (walletId: WalletId) => {
+  const timestamp30DaysAgo = new Date(Date.now() - MS_PER_30_DAYS)
+  const activenessThreshold = USER_ACTIVENESS_MONTHLY_VOLUME_THRESHOLD
+
+  const ledger = LedgerService()
+  const volume = await ledger.allTxVolumeSince({
+    timestamp: timestamp30DaysAgo,
+    walletId,
+  })
+  if (volume instanceof Error) return volume
+
+  return (
+    volume.outgoingSats > activenessThreshold || volume.incomingSats > activenessThreshold
+  )
+}
+
+export const getRecentlyActiveAccount = async (): Promise<
```

Comment:
Can we have the public (ie `export`ed functions) above the private ones in the file?
IMO its easier to understand quickly what the most important entry points of a module are if they are at the top. I think we discussed this before - but if not we can bring it up at code review this evening to gain consensus.

Reply:
will do. I'm just used to do the opposite